CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+-
+
+### Changed
+
+-
+
+### Fixed
+
+-
+
+## [0.1.0] - 2024-10-16
+
+- Initial release
```

Comment:
Should we add here everything?

Reply:
We could... but not sure if it's worth it. I would focus more on being disciplined from this point onwards. I remember finding this online "While you don't need to provide detailed information about the first release, it's good to acknowledge it" and that's the approach I followed